docs/sources/tutorials/grafana-fundamentals/index.md

```diff
@@ -42,11 +49,25 @@ Alternatively, you can also watch our Grafana for Beginners series where we disc
 - [Docker Compose](https://docs.docker.com/compose/) (included in Docker for Desktop for macOS and Windows)
 - [Git](https://git-scm.com/)
 
-### KillerCoda sandbox environment (Alternative)
+{{% /class %}}
 
-If you would prefer to follow along with this tutorial without needing to set up a local environment, you can use the [KillerCoda sandbox environment](https://killercoda.com/grafana-labs/course/full-stack/tutorial-enviroment).
+<!-- INTERACTIVE ignore START -->
 
-{{% /class %}}
+{{< admonition type="tip" >}}
+Alternatively, you can try out this example in our interactive learning environment: [Get started with Grafana Alerting](https://killercoda.com/grafana-labs/course/full-stack/tutorial-enviroment).
```

Comment:
I think the url needs to be changed to match where you have placed the tutorial:
```
grafana/grafana-fundamentals
```
also the description

Reply:
done!
which description ?

Reply:
it currently says
`Get started with Grafana Alerting` wondered if that was correct